### PR TITLE
Various changes to the enchanted boltaction rifle and its subtypes.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Enchanted bolt rifle:
Will no longer hit the user upon discarding. Sadly it lost its unique discard message in the process. Functionality over flavour, I guess. :sob:

Arcane Barrage:
Will no longer display a throw message when thrown. Instead it will silently be discarded.
Now has the NO_BLUDGEON flag

Blood barrage:
Has a new projectile sprite that looks (slightly) less out of place.
The projectile sprite will no longer be rotated.
It will no longer display magical sparks when shooting. Instead it will display cult sparks.

And most importantly, the latter two of them will no longer runtime to hell because they got deleted before the shot got completed.

## Why It's Good For The Game

Fixes some bugs and makes blood barrage more flavourful.

I know it's not as atomic as it could be, but most of these changes are cosmetic/bug fixes. Please yell at me if it's a big problem.

## Changelog
:cl:
fix: The enchanted bolt action rifle
imageadd: Blood barrage has a new projectile icon.
code: Fixed an arcane barrage runtime
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult_items.dm [hunk 1]
```diff
@@ -763,18 +763,19 @@
 	guns_left = 24
 	mag_type = /obj/item/ammo_box/magazine/internal/boltaction/enchanted/arcane_barrage/blood
 	fire_sound = 'sound/magic/wand_teleport.ogg'
-	item_flags = NEEDS_PERMIT | NOBLUDGEON | DROPDEL
```
Comment (4dplanner) [inline code/modules/antagonists/cult/cult_items.dm]: ?

**Comment by Tlaltecuhtli** [comment]
the whole point of it was the anime reference why would you remove the unique message

**Comment by nemvar** [comment]
Because the unique message was only possible because the item skipped sanity. That's why. And the sane solution already generates a message.

"[user] tosses aside the spent rifle!" is a shitty reference, if it even is one. Sorry that the snowflake message gets replaced with a more generic reference. *really* sorry.

**Comment by XDTM** [comment]
A middle ground option would be renaming the gun to "spent rifle" before the throw

**Pull request merged**